src/Relude/String/Conversion.hs

```diff
@@ -177,6 +178,32 @@ instance ConvertUtf8 LText LByteString where
     decodeUtf8Strict = LT.decodeUtf8'
     {-# INLINE decodeUtf8Strict #-}
 
+instance ConvertUtf8 String ShortByteString where
+    encodeUtf8 :: String -> ShortByteString
+    encodeUtf8 = toShort . T.encodeUtf8 . T.pack
```

Comment:
What do you think about reusing `ConvertUtf8` instance for `String ByteString` here? Like:

```haskell
encodeUtf8 = toShort . encodeUtf8 @String @ByteString  -- or vice versa
```